crates/hir_ty/src/lib.rs

```diff
@@ -1053,13 +1064,15 @@ mod tests {
         check_in_root_file(
             r#"
                 fn main() {
-                    let a = true
+                    let a = true;
+                    let mut b = 10;
```

Comment:
**Performance Issue**: Introduction of mutable variable 'b' which may lead to unnecessary data copying impacting performance, especially if used in a loop or called multiple times. <br> **Fix**: Evaluate the necessity of mutability for 'b'. If the variable does not need to be mutated, remove the 'mut' keyword to potentially enhance performance. <br> **Code Suggestion**: 
 ```
 -                    let mut b = 10;
 +                    let b = 10;
 ```